### PR TITLE
neutron-l3 role is renamed to neutron-network

### PR DESCRIPTION
**Pull request by jsuchome**
Neutron roles are renamed in https://github.com/crowbar/barclamp-neutron/pull/210

### CODE REVIEW

**Comment by vuntz** [comment]
Hrm, that'll be tricky: we need to only do that for Cloud 5 with updates (and 6+).

**Comment by jsuchome** [comment]
Ha. So, iscloudver 5plus is not enough righ? How do we distinguish between Cloud5 with and without updates?

**Comment by jsuchome** [comment]
susecloud5 source should have the update repo added, and develcloud5 should have the latest Cloud5 stuff, so I believe ``iscloudver 5plus`` check here should work...

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1723,16 +1723,20 @@ function custom_configuration()
...
+                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['$sle12node']"
             fi
 
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-server']" "['cluster:$clusternamenetwork']"
-                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-l3']" "['cluster:$clusternamenetwork']"
+                if iscloudver 5plus ; then
+                    proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['cluster:$clusternamenetwork']"
+                else
+                    proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-l3']" "['cluster:$clusternamenetwork']"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: could it break on GM5 without updates? then use proposal_set_value ...-network || proposal_set_value -l3

**Comment by jdsn** [comment]
Please squash 04230f0 into b003185. It does not make sense to add broken code and remove it later.

**Comment by vuntz** [comment]
@jsuchome can you rebase this PR?

**Comment by jsuchome** [comment]
@vuntz done

**Pull request merged**